com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/util/SpecialResourceTypesRequiredParametersUtil.java

```diff
@@ -0,0 +1,25 @@
+package com.sap.cloud.lm.sl.cf.core.util;
+
+import java.util.Map;
+import java.util.Set;
+
+import com.sap.cloud.lm.sl.cf.core.activiti.Messages;
+import com.sap.cloud.lm.sl.cf.core.cf.v1.ResourceType;
+import com.sap.cloud.lm.sl.common.ContentException;
+
+public class SpecialResourceTypesRequiredParametersUtil {
+
+    public static void checkRequiredParameters(ResourceType resourceType, Map<String, Object> parameters) {
+        Set<String> requiredParameters = getRequiredParameters(resourceType);
+        for (String parameter : requiredParameters) {
+            if (!parameters.containsKey(parameter)) {
+                throw new ContentException(Messages.NO_VALUE_GIVEN_FOR_REQUIRED_PARAMETERS);
```

Comment:
Add the name of the missing parameter in the error message. That way the user will know exactly what he's missing.